scripts/template.html

```diff
@@ -0,0 +1,138 @@
+        ];
+        */
+        var view = null;
+        var currentWorld = null;
+
+        function showWorld(worldName) {
+            if (view) {
+                view.resetSimulation();
+                view.destroyWorld();
+                view.close();
+                view = null;
+                document.getElementById("view3d").innerHTML = null;
+            }
+            if (!view) {
+                view = new webots.View(document.getElementById("view3d"));
+            }
+            
+            view.open(worldName + ".x3d");
+            view.setAnimation(worldName + ".json", "play", true);
+        }
+
+        function onWorldClicked(e, worldName) {
+            e.preventDefault();
+            window.location.replace('#' + worldName);
+        }
+
+        function createSidebarItems() {
+            const sidebar = document.getElementById('sidebar');
+            for (const world of WORLD_LIST) {
+                sidebar.innerHTML += `
+                    <div class="simulation-item">
+                        <a href="javascript:void(0)" onclick="onWorldClicked(event, '${world.name}')">
+                            <h3>${world.title} (${world.name})</h3>
+                        </a>
+                        <p>${world.description}</p>
+                        <a href="${world.name}.html" target="_blank">[extern]</a>
+                    </div>
+                `;
+            }
+        }
+
+        function getWorldFromHash() {
+            if (window.location.hash) {
+                const worldName = window.location.hash.substring(1);
+                const matchingWorld = WORLD_LIST.find(element => element.name == worldName);
+                if (matchingWorld) {
+                    return matchingWorld;
+                }
+                return null;
+            }
+            return null;
+        }
+
+        function showWorldFromHash() {
+            currentWorld = getWorldFromHash();
+            if (!currentWorld) {
+                window.location.replace('#' + WORLD_LIST[0].name);
+                return;
+            }
+            console.log(currentWorld);
+            showWorld(currentWorld.name);
+        }
+
+        function init() {
+            createSidebarItems();
+            showWorldFromHash();
+        }
+
+        function hashChanged() {
+            showWorldFromHash();
+        }
+
+        window.addEventListener("load", init, false);
+        window.addEventListener('hashchange', hashChanged);
+
+        if (location.protocol == "file:" && (!!window.chrome && !!window.chrome.webstore))
+            alert("Webots HTML5 Models and Animations cannot be loaded locally on Google Chrome, as Chrome does not support cross-origin requests using the file:// protocol.");
+    </script>
+</head>
+
+<body>
+    <div id="view3d"></div>
+    <div id="sidebar">
+        <h2>List of simulations</h2>
+    </div>
+</body>
+
+</html>
```

Comment:
Missing final line feed.
```suggestion
</html>

```